ui/components/package.json

```diff
@@ -57,5 +57,8 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "typedoc": {
+    "entryPoint": "./src/index.ts"
+  }
```

Comment:
This annotation tells typedoc to look at the TypeScript entry point. By default, it will try to look at whatever is defined in `main`, which is the built code. It will throw this error if it uses that.

> error Entry point "/Users/juliepagano/src/perses/perses/ui/components/dist/cjs/index.js" does not appear to be built by/included in the tsconfig found at "/Users/juliepagano/src/perses/perses/ui/components/tsconfig.json"